app/controllers/indoor_controller.rb

```diff
@@ -0,0 +1,27 @@
+class IndoorController < ApplicationController
+  def demo
```

Comment:
The demo action should only stay in code if you think it's neccessary for other teams to understand it